.eslintignore

```diff
@@ -1,9 +1,7 @@
 node_modules
-lib/internal/v8.js
 lib/internal/v8_prof_polyfill.js
```

Comment:
I am not sure what this file does. Do we have to ignore the file here?

Reply:
My vague recollection: I think the file was originally copied from either V8 source or a V8 change proposal that may have been rejected by V8. Since we were copying the file, we didn't apply our lint style to it since we would just copy changes from upstream at a later date, presumably. This is similar, I think, to how we handle punycode.js. It looks like we may now basically own v8_prof_polyfill.js, though. I'm not sure. @matthewloring may know,.